anomaly_detection/utils/np_json.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2019 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import base64
+import json
+
+import numpy as np
```

Comment:
numpy is very short.  I'd prefer not to shorten it to "np".

Reply:
This is a habitual usage，actually many examples use the shorter one.

Reply:
Not a big deal, but "np" means "no problem" for me, so it sounds odd:).